x/data/server/testsuite/suite.go

```diff
@@ -210,3 +216,41 @@ func (s *IntegrationTestSuite) TestRawDataScenario() {
 	require.NotNil(queryRes)
 	require.Equal(ts, queryRes.Entry.Timestamp)
 }
+
+func (s *IntegrationTestSuite) TestResolver() {
```

Comment:
should we be checking more here? i.e. checking query responses are equal to msg server requests

Reply:
https://github.com/regen-network/regen-ledger/pull/708/commits/a7fc3fc42d340d349618539d8f407102ab19a470